Why Elm.md

```diff
@@ -2,32 +2,41 @@
 
 In this section, we'll talk about the advantages — and disadvantages! — of using Elm to make front-end applications.
 
-First, the basics: **Elm is a language for building complex web interfaces.** Much like CoffeeScript or JSX, Elm compiles to JavaScript; the Elm Platform takes your Elm code and turns it into JavaScript that any web browser can understand. In many ways, it operates similarly to front-end JavaScript frameworks like React, Angular, and Ember.
 
-So why do developers go to the trouble of using a whole new language instead of using a JavaScript framework? Indeed, Elm shares many of its opinions about architecture and code quality with those tools. However, **Elm is different in one important way — because it is a distinct programming language and not simply a framework, Elm can strictly enforce its opinions.**
+## First off, what is Elm?
 
-And the language has lots of rules! They are meant to push developers toward writing code that is easier to maintain and harder to break. For instance:
+**Elm is a language for building complex web interfaces.** Much like CoffeeScript or JSX, Elm compiles to JavaScript; the Elm Platform takes your Elm code and turns it into JavaScript that any web browser can understand. In many ways, it operates similarly to front-end JavaScript frameworks like React, Angular, and Ember.
 
-- **In Elm, a function always returns the same value for a given input.** There is no notion of instance variables or `this` in Elm — the only thing a function can do is read the values of its arguments and call out to other functions. This quality of the language is called **purity** — no matter what context, a function will do the same thing every time.
+Elm is more powerful than a frontend framework though: **it is a distinct programming language that can strictly enforce its opinions.**
 
-- **Once a value is declared in an Elm application, it can never change.** It's not possible, for instance, for a function to accidentally change its inputs, or for something to become `undefined`. If you need to add an element to a list, there is no way to do so destructively — instead, you have to create a new list with all the elements you specify. Values like these are called **immutable**.
+You may have heard words like **functional** and **statically typed** used to describe Elm. **Functional** means that a function always returns the same value for a given input. **Statically typed** means that the compiler checks that the code makes sense before you can use it in production: it won't let you ask for the length of an integer or try to multiply a string and a float. If your code compiles, that means that you're doing operations that make sense. If you give a function the wrong *type* of value (a `String` rather than an `Int`) the Elm compiler will figure it out immediately and give you a hint. (This also means that a function must always return the same *type* of value. Try to think through why this is the case.)
```

Comment:
Love this rewrite! short and to the point. 

Only suggestion: let's remove the final parenthetical, since it's pretty far away from the message of this lesson.